.github/workflows/ci.yml

```
@@ -27,3 +28,5 @@ jobs:
         sudo ./startup
     - name: Run Tests
       run: go test -v startup_test.go
+      env:
+        DISCORD_KEY: ${{ secrets.SINUSBOT_DISCORD_CI }}
```

```suggestion
        DISCORD_API_KEY: ${{ secrets.SINUSBOT_DISCORD_API_KEY }}
```

that makes it more clear